akka-stream-tests/src/test/scala/akka/stream/scaladsl/ActorRefBackpressureSourceSpec.scala

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.scaladsl
+
+import akka.actor.Status
+import akka.stream.testkit.Utils.TE
+import akka.stream.testkit.scaladsl.StreamTestKit.assertAllStagesStopped
+import akka.stream.testkit.scaladsl.TestSink
+import akka.stream.{ ActorMaterializer, Attributes }
+import akka.stream.testkit.StreamSpec
+import akka.testkit.TestProbe
+
+import scala.concurrent.duration._
+
+private object ActorRefBackpressureSourceSpec {
+  case object AckMsg
+}
+
+class ActorRefBackpressureSourceSpec extends StreamSpec {
+  import ActorRefBackpressureSourceSpec._
+
+  private implicit val materializer: ActorMaterializer = ActorMaterializer()
+
+  "An ActorRefBackpressureSourc" must {
```

Comment:
missing `e` here, but I think using the factory name is easier in the test output, so could be `"Source.actorRefWithAck" must` instead